src/components/queryEditor.tsx

```diff
@@ -311,11 +311,7 @@ export function QueryEditor(props: EditorProps) {
   }, [tab]);
 
   const tabId = (t) => {
-    if (
-      t === Tabs.Templates ||
-      t === Tabs.FlexibleDataModelling ||
-      t === Tabs.ExtractionPipelines
-    ) {
+    if (t === Tabs.Templates || t === Tabs.FlexibleDataModelling) {
```

Comment:
this is something I cannot agree with. extraction pipelines must stay in preview until we gather more feedback.
it doesn't matter if it's stable API or not. in the connector it's still an experimental feature